### PR TITLE
Chore: Optimize deployment scripts

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: scripts/build-manifest.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Remove previous subgraph if there is any
+if [ -f subgraph.yaml ]; then
+  echo 'Removing previous subgraph manifest...'
+  rm subgraph.yaml
+fi
+
+# Build subgraph manifest for requested variables
+echo "Preparing new subgraph for Court address ${COURT} to network ${NETWORK}"
+cp subgraph.template.yaml subgraph.yaml
+sed -i -e "s/{{network}}/${ENV}/g" subgraph.yaml
```
Comment (sohkai) [inline scripts/build-manifest.sh:66]: Small note, perhaps we can align how we template across our subgraphs. I believe the `dao-subgraph` is using mustache (and/or we may want to use [jsonnet](https://jsonnet.org/) instead).

File: subgraph.template.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ dataSources:
     source:
       address: '{{court}}'
       abi: AragonCourt
+      startBlock: {{startBlock}}
```
Comment (sohkai) [inline subgraph.template.yaml:13]: Nice, this is just to make the initial sync faster, right?

File: scripts/build-manifest.sh (hunk 1 above)
Comment (facuspagnuolo) [reply scripts/build-manifest.sh:66]: totally, let's align these :)

File: subgraph.template.yaml (hunk 2 above)
Comment (facuspagnuolo) [reply subgraph.template.yaml:13]: yea 🎉

**Pull request merged**